.github/workflows/pr.yml

```diff
@@ -0,0 +1,58 @@
+name: Runs tests
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+  release:
+  workflow_dispatch:
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: vmq_test_user
+          POSTGRES_PASSWORD: vmq_test_password
+          POSTGRES_DB: vmq_test_database
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      memcached:
+        image: memcached
+        ports:
+          - 11211:11211
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      redissentinel:
+        image: bitnami/redis-sentinel
+        ports:
+          - 26379:26379
+        options: >-
+          --health-cmd "redis-cli -p 26379 ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2.0.0
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: 23.0
+      - run: sudo apt-get -y install libsnappy-dev
+      - run: sudo apt-get -y install pcregrep
+      - run: pwd
+      - run: sh ./run-tests-with-retry.sh .
```

Comment:
Add empty line at the EOF

Reply:
Should we add linter to detect such issue?

Reply:
Added empty line

Reply:
Github is not showing the new lines for some reason. It does not show them on any file.